server/src/immich/api-v1/asset/dto/import-asset.dto.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsOptional, IsString, IsBoolean } from 'class-validator';
+import fs from 'fs/promises';
+import path from 'path';
+import crypto from 'crypto';
+import mime from 'mime-types';
+import { CreateAssetDto } from './create-asset.dto';
+
+export class ImportAssetDto extends CreateAssetDto {
+  @IsOptional()
+  @IsBoolean()
+  isReadOnly?: boolean = true;
+
+  @ApiProperty()
+  @IsString()
+  assetData!: string;
+
+  @ApiProperty()
+  @IsString()
+  @IsOptional()
+  livePhotoData?: string;
+
+  @ApiProperty()
```

Comment:
These API property decorators are not needed

Reply:
Removed these.